locations/spiders/redtag_fashion_ae_sa.py

```
@@ -0,0 +1,104 @@
+            item["city"] = store.get("city")
+            item["country"] = store.get("country")
+
+            # Hours
+            hours = store.get("timming")  # Note the misspelling in the API
+            if hours:
+                item["opening_hours"] = self.parse_hours(hours)
+
+            # Additional properties
+            item["extras"] = {
+                "shop": "clothes",
+            }
+            if store.get("homeware") == "1":
+                item["extras"]["homeware"] = True
+            if store.get("cosmetics") == "1":
+                item["extras"]["cosmetics"] = True
+
+            yield item
+
+    def parse_hours(self, hours_str: str) -> str:
+        """Parse hours string into OpenStreetMap format"""
+        try:
+            if not hours_str:
+                return None
+
+            formatted_hours = []
+
+            # Split different day ranges
+            for section in hours_str.split("*"):  # Changed from | to * based on actual data
+                section = section.strip()
+                if not section:
+                    continue
+
+                # Split days and times
+                days, times = section.split(" ", 1)
+                days = days.strip()
+                times = times.strip()
+
+                # Handle "Sun to Thu" format
+                if "to" in days:
+                    start_day, end_day = days.split(" to ")
+                    days = f"{start_day}-{end_day}"
+                # Handle "Fri & Sat" format
+                elif "&" in days:
+                    days = days.replace(" & ", ",")
+
+                # Clean up times
+                times = times.replace(" AM", ":00") if "AM" in times else times.replace(" PM", ":00")
+
+                formatted_hours.append(f"{days} {times}")
+
+            return "; ".join(formatted_hours)
+        except Exception as e:
+            self.logger.warning(f"Could not parse hours: {hours_str} - {e}")
+            return None
```

Instead of wrapping the body of `parse_hours()`, let the exception bubble out of this function and catch it where you call `parse_hours()`.